### DISCUSSION TITLE
Option to suppress the Fan Control window from popping up when errors occur

### DISCUSSION THREAD

**Discussion by CaveWaverider** [Rem0o/FanControl.Releases#1678] [Ideas]
Fan Control is great. However, when I have it set on Windows and "Start minimized" is enabled, sometimes Fan Control's Window will pop up saying sensors are missing...retrying, after a second or so of retrying all sensors seem to be detected and it works fine, but the window popping up is annoying.
I have tried to set a startup delay as suggested and tried up to 3 minutes, but it still seems to occur. My Mainboard is an ASUS ROG STRIX Z690-A GAMING WIFI D4.

Would it be possible to add an option to suppress the Fan Control Window from popping up when any errors occur, especially when Fan Control is set to be "Start minimized"?

Thanks in advance!

**Comment by Rem0o** [comment]
Yes, but then FanControl silently won't be working, which can be an issue.

Add the "-m" command line argument, it overrides everything.

**Comment by CaveWaverider** [comment]
Since version V155 the window pops up every time on windows startup (without any errors) when I'm starting it with the "-m" command. Previously, it succeeded in suppressing it. Perhaps it has to do with the patch note "Fix an instance where the main window would pop up on start", although for me it does the opposite than intended.

Update: Going back to V154 fixes the issue.
